rholang/src/main/scala/coop/rchain/rholang/interpreter/Runtime.scala

```diff
@@ -78,21 +78,23 @@ object Runtime {
   type Remainder = Option[Var]
   type BodyRef   = Long
 
-  class BlockDataStorage[F[_]](val timestamp: Ref[F, Par]) {
-    def setParams(timestamp: Par)(implicit F: Sync[F]): F[Unit] =
-      for {
-        _ <- this.timestamp.set(timestamp)
-      } yield ()
+  final case class BlockData(timeStamp: Long, blockNumber: Long)
+
+  class BlockDataStorage[F[_]](storage: Ref[F, Option[BlockData]]) {
```

Comment:
Why is it an `Option`?

Reply:
The original implementation was storing Par and the initial value for timestamp was Par(). This was the value until setParams was first called. 

In this PR I changed the underlying storage to Long so I used Option, because I don't have a value to store before the first setParams is called